### DISCUSSION TITLE
Mute per channel - Mute Button per channel

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#2616] [Ideas]
I would like to mute "only" my microphone on for example "channel 1 L" but continue to send "channel 2 R" guitar

Possible to a add a mute button per channel?

<img width="205" alt="grafik" src="https://user-images.githubusercontent.com/77500932/165284594-72b28a16-ba5b-4b1b-b0bc-a1c77a647b6f.png">

Thanks
Reinhard

**Comment by ann0see** [comment]
You can use the pan/balance setting in Advanced settings. I think it was even developed for this use case or something similar
